features/src/main/feature/camel-features.xml

```diff
@@ -1617,42 +1617,36 @@
-<!--        <bundle dependency='true'>mvn:org.codehaus.woodstox/stax2-api/${stax2v4-api-bundle-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:com.fasterxml/aalto-xml/0.9.10</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-common/${netty-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-transport/${netty-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-transport-native-unix-common/${netty-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-buffer/${netty-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-handler/${netty-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-resolver/${netty-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-codec/${netty-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-codec-http/${netty-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-tcnative-classes/${netty-tcnative-classes-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.apache.camel/camel-olingo4-api/${project.version}</bundle>-->
-<!--        <bundle>mvn:org.apache.camel/camel-olingo4/${project.version}</bundle>-->
-<!--    </feature>-->
+    <feature name='camel-olingo4' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <feature version="[4,5)">http-client</feature>
+        <feature>jakarta-servlet</feature>
+        <feature>jackson</feature>
+        <bundle dependency='true'>mvn:org.apache.olingo/odata-commons-api/5.0.0</bundle>
+        <bundle dependency='true'>mvn:org.apache.olingo/odata-commons-core/5.0.0</bundle>
+        <bundle dependency='true'>mvn:org.apache.olingo/odata-client-api/5.0.0</bundle>
+        <!-- TODO remove wrap protocol when https://issues.apache.org/jira/browse/OLINGO-1206 is resolved -->
+        <bundle dependency='true'>wrap:mvn:org.apache.olingo/odata-server-api/5.0.0$overwrite=merge&amp;Export-Package=org.apache.olingo.server.api;version=5.0.0,org.apache.olingo.server.api.uri;version=5.0.0,org.apache.olingo.server.api.processor;version=5.0.0,org.apache.olingo.server.api.batch;version=5.0.0,org.apache.olingo.server.api.debug;version=5.0.0,org.apache.olingo.server.api.deserializer;version=5.0.0,org.apache.olingo.server.api.deserializer.batch;version=5.0.0,org.apache.olingo.server.api.etag;version=5.0.0,org.apache.olingo.server.api.prefer;version=5.0.0,org.apache.olingo.server.api.serializer;version=5.0.0,org.apache.olingo.server.api.uri.queryoption;version=5.0.0,org.apache.olingo.server.api.uri.queryoption.apply;version=5.0.0,org.apache.olingo.server.api.uri.queryoption.expression;version=5.0.0,org.apache.olingo.server.api.uri.queryoption.search;version=5.0.0,org.apache.olingo.netty.server.api;version=5.0.0,org.apache.olingo.netty.server.core;version=5.0.0</bundle>
```

Comment:
Could be simplified using the character *, I mean something like `Export-Package=org.apache.olingo.server.*;version=5.0.0`

Reply:
done, 14 packages to one, thanks!